core/src/main/java/io/opencensus/tags/TagContext.java

```diff
@@ -37,9 +37,12 @@
 @Immutable
 public abstract class TagContext {
 
-  // TODO(sebright): Consider removing TagContext.unsafeGetIterator() so that we don't need to
-  // support fast access to tags.
-  public abstract Iterator<Tag> unsafeGetIterator();
+  /**
+   * Returns an iterator over the tags in this {@code TagContext}.
+   *
+   * @return an iterator over the tags in this {@code TagContext}.
+   */
+  protected abstract Iterator<Tag> iterator();
```

Comment:
Copy rationale note about protected here to reduce chance of misunderstanding later.

Reply:
Done